core/common/src/main/java/alluxio/AlluxioURI.java

```diff
@@ -11,20 +11,21 @@
 
 package alluxio;
 
+import alluxio.util.URIUtils;
+
 import org.apache.commons.lang.StringUtils;
 
 import java.io.Serializable;
-import java.net.URI;
-import java.net.URISyntaxException;
 import java.util.Arrays;
+import java.util.Map;
 
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * It uses a hierarchical URI internally. URI requires that String is escaped, {@link AlluxioURI}
- * does not.
+ * It uses a {@link URI} internally. URI requires that String is escaped, {@link AlluxioURI} does
```

Comment:
Please add a sentence explaining the purpose for this class, before jumping into how it works internally.